main_exercise10.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+from exercise10 import (
+    get_price_quantity,
+    get_subtotal,
+    calculate_sum_of_subtotals,
+    calculate_total_tax_for_items,
+    calculate_grand_total,
+)
+
+TAX_RATE = 5.5
+
+def main():
+    """Create a self-checkout system that calculates the grand total for three items"""
+
+    price_item1, quantity_item1 = get_price_quantity()
+    print("\n")
+    price_item2, quantity_item2 = get_price_quantity()
+    print("\n")
+    price_item3, quantity_item3 = get_price_quantity()
+
+    subtotal_item1 = get_subtotal(price_item1, quantity_item1)
+    subtotal_item2 = get_subtotal(price_item2, quantity_item2)
+    subtotal_item3 = get_subtotal(price_item3, quantity_item3)
+
+    sum_of_subtotals = calculate_sum_of_subtotals(subtotal_item1, subtotal_item2, subtotal_item3)
+    total_tax_for_items = calculate_total_tax_for_items(TAX_RATE, sum_of_subtotals)
```

Comment:
Excellent interface (what the function takes in as input) for `calculate_total_tax_for_items`. That's exactly what I'd do. However, one nit-pick is to reverse the parameter order as the subtotal is a bit more important imo that the tax rate. The more important parameters should be listed first.